src/main/java/org/broadinstitute/consent/http/service/ConsentService.java

```diff
@@ -1,69 +1,36 @@
-import javax.inject.Inject;
-import javax.ws.rs.NotFoundException;
-import javax.ws.rs.WebApplicationException;
-import javax.ws.rs.core.Response;
-import org.apache.commons.collections.CollectionUtils;
 import org.apache.commons.lang3.StringUtils;
 import org.broadinstitute.consent.http.db.ConsentDAO;
-import org.broadinstitute.consent.http.db.DataAccessRequestDAO;
-import org.broadinstitute.consent.http.db.DatasetDAO;
 import org.broadinstitute.consent.http.db.ElectionDAO;
-import org.broadinstitute.consent.http.db.VoteDAO;
-import org.broadinstitute.consent.http.enumeration.AuditActions;
-import org.broadinstitute.consent.http.enumeration.AuditTable;
 import org.broadinstitute.consent.http.enumeration.DataUseTranslationType;
 import org.broadinstitute.consent.http.enumeration.ElectionType;
 import org.broadinstitute.consent.http.exceptions.UnknownIdentifierException;
-import org.broadinstitute.consent.http.models.AuthUser;
 import org.broadinstitute.consent.http.models.Consent;
-import org.broadinstitute.consent.http.models.ConsentAssociation;
-import org.broadinstitute.consent.http.models.Dataset;
 import org.broadinstitute.consent.http.models.Election;
-import org.jdbi.v3.core.Handle;
-import org.jdbi.v3.core.Jdbi;
-import org.jdbi.v3.core.generic.GenericType;
-import org.jdbi.v3.core.statement.PreparedBatch;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import javax.inject.Inject;
+import javax.ws.rs.NotFoundException;
+import java.sql.Timestamp;
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
 public class ConsentService {
 
-    private final AuditService auditService;
-    private final Jdbi jdbi;
     private final Logger logger;
-    private final DatasetDAO dataSetDAO;
-
     private final ConsentDAO consentDAO;
     private final ElectionDAO electionDAO;
-    private final VoteDAO voteDAO;
-    private final DacService dacService;
-    private final DataAccessRequestDAO dataAccessRequestDAO;
     private final UseRestrictionConverter useRestrictionConverter;
 
     @Inject
-    public ConsentService(ConsentDAO consentDAO, ElectionDAO electionDAO, VoteDAO voteDAO, DacService dacService,
-                          DataAccessRequestDAO dataAccessRequestDAO, AuditService auditService,
-                          Jdbi jdbi, DatasetDAO dataSetDAO,
+    public ConsentService(ConsentDAO consentDAO, ElectionDAO electionDAO,
                           UseRestrictionConverter useRestrictionConverter) {
         this.consentDAO = consentDAO;
         this.electionDAO = electionDAO;
-        this.voteDAO = voteDAO;
-        this.dacService = dacService;
-        this.dataAccessRequestDAO = dataAccessRequestDAO;
-        this.auditService = auditService;
-        this.jdbi = jdbi;
-        this.dataSetDAO = dataSetDAO;
         this.useRestrictionConverter = useRestrictionConverter;
         this.logger = LoggerFactory.getLogger(this.getClass());
     }
```

Comment:
the function below this, getById, appears that it can be deleted (not used anywhere but 1 test)

Reply:
Nice catch